### PR TITLE
Fix a translation error

### PR DESCRIPTION
**Pull request by KDr2**
And incidentally, fixed an emacs settings error.

### CODE REVIEW

**Comment by alissa-tung** [comment]
Should we merge this [change](https://github.com/plfa/plfa.github.io/commits/dev/dotdir-locals.el) btw?

**Comment by KDr2** [comment]
> Should we merge this [change](https://github.com/plfa/plfa.github.io/commits/dev/dotdir-locals.el) btw?

Oh, I didn't know that commit before you mentioned it, but https://github.com/plfa/plfa.github.io/issues/516 was the same issue I found when I was reading the book in my emacs. So after some debugging, I found the issue in `.dir-local.el` and managed to find a way to fix it.

I think https://github.com/plfa/plfa.github.io/commit/463361eb1313f2c48716556f47b15a5241ae5b42 is a workaround when one wasn't familiar with emacs, it will be better to port the change of that file to the original repo :slightly_smiling_face:

File: src/plfa/part1/Relations.lagda.md [hunk 1]
```diff
@@ -287,7 +287,7 @@ either the left or right, as it makes no sense to parse `1 ≤ 2 ≤ 3` as
 either `(1 ≤ 2) ≤ 3` or `1 ≤ (2 ≤ 3)`.
 -->
 
-我们将 `_≤_` 的优先级设置为 4，所以它比优先级为 6 的 `_+_` 结合的更紧，此外，
+我们将 `_≤_` 的优先级设置为 4，所以它不如优先级为 6 的 `_+_` 结合地紧，因而，
```
Comment (starxingchenc) [inline src/plfa/part1/Relations.lagda.md]: 应为 “结合得紧”

**Review by starxingchenc — CHANGES_REQUESTED** [review]

File: src/plfa/part1/Relations.lagda.md (hunk 1 above)
Comment (KDr2) [reply src/plfa/part1/Relations.lagda.md]: :+1: updated.

**Review by starxingchenc — APPROVED** [review]

**Review by fangyi-zhou — CHANGES_REQUESTED** [review]
Please remove the changes regarding emacs. It would be nice if you can upstream this change and we can pull this from upstream

**Comment by KDr2** [comment]
> Please remove the changes regarding emacs. It would be nice if you can upstream this change and we can pull this from upstream

OK, updated.

**Pull request merged**